rust-runtime/aws-smithy-http-server/src/plugin/mod.rs

```diff
@@ -111,6 +111,8 @@ mod filter;
 mod identity;
 mod layer;
 mod pipeline;
+#[doc(hidden)]
```

Comment:
Why hidden?

Reply:
I'd like `Scoped` and the `scope` macros to be the primary API and the `#[doc(hidden)]` items to be implementation details.